resources/ts/Components/Card.tsx

```diff
@@ -8,28 +8,33 @@ type Props = {
   title: ReactElement | string;
   description?: string;
   img?: string;
+  badge?: string;
   active?: boolean;
   contentStyles?: string;
   textCenter?: boolean;
+  border?: boolean;
 } & Omit<ComponentProps<"div">, "title">;
 
 export const Card = ({
   title,
   description,
   img,
+  badge,
   active,
   children,
   className,
   contentStyles,
+  border,
```

Comment:
The props that are optional and boolean like `border` and `badge` better have a explicit false default value